src/main.cpp

```diff
@@ -233,8 +233,14 @@ void destroy() {
 		throw std::runtime_error("/usr/bin/tldr could not be removed");
 	}
 	std::cout << "Removed executable /usr/bin/tldr" << std::endl;
-
-	string cache_dir = "/home/" + string(su) + "/.tldr";
+	
+	string cache_dir = "";
```

Comment:
That's only necessary for primitive datatypes. Objects like strings or vectors will be empty by default.
```suggestion
	string cache_dir;
```